pilot/pkg/model/push_context.go

```diff
@@ -429,6 +429,13 @@ var (
 		"Duplicate envoy clusters caused by service entries with same hostname",
 	)
 
+	// StaticNoEndpointClusters tracks dns/static clusters without endpoints
+	StaticNoEndpointClusters = monitoring.NewGauge(
+		"pilot_dns_cluster_without_endpoints_envoy_clusters",
```

Comment:
pilot_dns_cluster_without_endpoints, actually we donot sum up ths static cluster.

Reply:
why this is back

Reply:
sorry，it is a mistake. fixed already